src/coreclr/jit/llvm.cpp

```diff
@@ -2392,6 +2458,48 @@ void Llvm::PlaceAndConvertShadowStackLocals()
     lowerToShadowStack();
 }
 
+static bool placeLocalInAlloca(LclVarDsc* varDsc)
+{
+    return !varDsc->lvInSsa;
+}
```

Comment:
```suggestion
static bool placeLocalInAlloca(LclVarDsc* varDsc)
{
    assert(canStoreLocalOnLlvmStack(varDsc) && (_compiler->fgSsaPassesCompleted >= 2));
    return !varDsc->lvInSsa;
}
```
We should not be asking this question for shadow stack locals, we also should not be asking it before codegen.

(Would be nice to start thinking about separating codegen into its own class)

Reply:
Ok, when you say codegen, it refers to anything used once we get to the `Llvm::Compile` method?